Python/optimizer.c

```diff
@@ -652,11 +652,6 @@ translate_bytecode_to_trace(
             RESERVE_RAW(2, "_EXIT_TRACE");
             max_length--;
         }
-        if (OPCODE_HAS_ERROR(opcode)) {
```

Comment:
Let's keep this in. Instead, let's add `OPCODE_HAS_DEOPT` check too like we discussed.